index.bs

```diff
@@ -10331,25 +10331,27 @@ Note: Remember that interface objects for callback interfaces only exist if they
 when they do exist, they are not [=function objects=].
 
 
-<h5 id="es-interface-call">Interface object \[[Call]] method</h5>
+<h5 id="es-constructible-interfaces" oldids="es-interface-call">Constructible Interfaces</h5>
 
-If the [=interface=] is declared with a
-[{{Constructor}}] [=extended attribute=],
-then the [=interface object=]
-can be called as a function to create an object that implements that
-interface.  Interfaces that do not have a constructor will throw
-an exception when called as a function.
+If the [=interface=] is declared with a [{{Constructor}}] [=extended attribute=],
+then the [=interface object=] can be called as a constructor
+to create an object that implements that interface.
+Calling that interface as a function will throw an exception.
 
-The internal \[[Call]] method
-of the interface object behaves as follows, assuming
-|arg|<sub>0..|n|−1</sub> is the list
-of argument values passed to the constructor, and |I|
-is the [=interface=]:
+Interfaces that do not have a constructor will throw
```

Comment:
Maybe "interfaces that are not declared with a [Constructor] extended attribute will throw when called, both as a function and as a constructor."